.github/workflows/build.yml

```diff
@@ -49,9 +56,39 @@ jobs:
           free -m
           docker system df
           docker system df -v
-      - name: Build EVE KVM
+
+  eve:
+    needs: packages
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: [arm64, amd64]
+        hv: [xen, kvm]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: update linuxkit cache if available
+        uses: actions/cache@v3
+        with:
+          path: ~/.linuxkit/cache
```

Comment:
Is it safe to clean cache inside Clean step in case of using this step? Will we end with cache cleanup while we potentially still need it in another runners?

Reply:
I don't see why not. Each job in a runner is completely independent. If you don't get rid of it, you will lose it anyways. Except on the persistent arm64 self-hosted runner, but that really should be ephemeral anyways.

Reply:
For now, it looks good. If this all passes, we merge it in. If it creates problems in the future, we worry about it then.

Reply:
I mean we have a step with `rm -rf ~/.linuxkit` in the end. But with new step you put here cache update will happen after cleaning (in post action). So my understanding here is if we will start one `packages` job for amd64-kvm but for some reason amd64-xen will delayed (for example we will hit limit of parallel runners), we will clean the cache before amd64-xen will be started. Correct me if I am wrong, as it depends on the internal logic of actions/cache.

Reply:
Each of those jobs runs in their own runner. So rm -rf on one runner has no impact on the other. It's why I wanted to use the matrix and run them in parallel, rather than the serial we had before.

That is for the linuxkit cache. The GitHub cache stays until it expires it, based on time or size.

Reply:
Got it, thanks for explaining!